src/Controls/src/BindingSourceGen/BindingCodeWriter.cs

```diff
@@ -40,7 +40,20 @@ private static bool ShouldUseSetter(BindingMode mode)
 		}
 		""";
 
-	private static string GenerateBindingCode(string bindingMethodBody) => $$"""
+	private static string GenerateUnsafeFieldAccessor(string fieldName, string memberType, string containingType, uint id) => $$"""
+		[UnsafeAccessor(UnsafeAccessorKind.Field, Name = "{{fieldName}}")]
+		private static extern ref {{memberType}} GetUnsafeField{{id}}{{fieldName}}({{containingType}} source);
```

Comment:
The method name is quit compilcated and we're building it in at least two places. I wonder if we should create a method `CreateUnsafeFieldAccessorMethodName(int bindingId, string fieldName)` to make sure they're always in sync or if it's not necessary (+ the same for the property).